frontend/src/app/app.component.css

```
@@ -22,6 +22,34 @@
+	font-size: 18px;
+	width: 18px;
+	height: 18px;
+}
+
+@media (width <= 600px) {
+	.main-menu-sidenav.mat-drawer {
+		top: 44px !important;
+		height: calc(100vh - 44px) !important;
+	}
+
+	.main-menu-container ::ng-deep .mat-drawer-backdrop.mat-drawer-shown {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)